service.subtitles.subtitle/service.py

```diff
@@ -125,11 +125,13 @@ def takeTitleFromFocusedItem():
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+    isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```

Comment:
@phil65 did you check what i`ve said?
The new code not affects regular addon usage.